config/experimental_config.yaml

```diff
@@ -0,0 +1,30 @@
+
+outputs:
+  experiment_name: default
+  save: false
+
+synthea:
+  population_num: '50'
+  county: 'West Yorkshire'
+
+generate:
+  llm_model_features:
+    llm_model_name: llama2
+    prompt_template_path: llama2_template.json
+
+extraction:
```

Comment:
Noting we have "structure" in some places, and "extraction" in others - should discuss and stick to one.